### PR TITLE
Remove outdated information about pulplift

### PR DESCRIPTION
**Pull request by lubosmj**

### CODE REVIEW

**Review by gerrod3 — DISMISSED** [review]
Can you revise the other inaccuracies in this file? It is pretty outdated looking at it.

**Comment by mdellweg** [comment]
I've recently seen a mention of `[noissue]` in the contributor section. While you are at it...

**Comment by lubosmj** [comment]
Middle ground: I just removed `[noissue]` occurrences. I am not planning to revise the test workflows in this PR.

File: docs/dev/guides/git.md [hunk 1]
```diff
@@ -42,9 +42,6 @@ Pope's blog post on the subject](http://tbaggery.com/2008/04/19/a-note-about-git
 
 Each commit message should link to an issue on the [pulpcore Github Issue tracker](https://github.com/pulp/pulpcore/issues/). See the [Github Linking Docs](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) and include at least one link in your commit message.
 
-If you must create a commit for which there is no issue, add the `[noissue]` syntax in the commit
-message.
```
Comment (mdellweg) [inline docs/dev/guides/git.md:46]: :1st_place_medal: Thank you!

**Review by mdellweg — APPROVED** [review]

**Pull request merged**